bundles/org.openhab.binding.freeathomesystem/src/main/java/org/openhab/binding/freeathomesystem/internal/util/PidTranslationUtils.java

```diff
@@ -0,0 +1,576 @@
+        mapDescObj.put("0xF10F", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_UNKNOWN, CATEGORY_UNDEFINED, "", "",
+                "pid-heartbeat-delay", "pid-heartbeat-delay-text"));
+        mapDescObj.put("0xF110", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_UNKNOWN, CATEGORY_UNDEFINED, "", "",
+                "pid-info-heartbeat-delay", "pid-info-heartbeat-delay-text"));
+        mapDescObj.put("0xFF01", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_UNKNOWN, CATEGORY_UNDEFINED, "", "",
+                "pid-measured-temperature-1", "pid-measured-temperature-1-text"));
+        mapDescObj.put("0xFF02", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_UNKNOWN, CATEGORY_UNDEFINED, "", "",
+                "pid-measured-temperature-2", "pid-measured-temperature-2-text"));
+        mapDescObj.put("0xFF03", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_UNKNOWN, CATEGORY_UNDEFINED, "", "",
+                "pid-measured-temperature-3", "pid-measured-temperature-3-text"));
+        mapDescObj.put("0xFF04", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_UNKNOWN, CATEGORY_UNDEFINED, "", "",
+                "pid-measured-temperature-4", "pid-measured-temperature-4-text"));
+        mapDescObj.put("0x061A", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_DECIMAL, CATEGORY_PRESSURE, "", "",
+                "pid-air-quality-pressure-value", "pid-air-quality-pressure-value-text"));
+        mapDescObj.put("0x061B", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_INTEGER, CATEGORY_UNDEFINED, "", "",
+                "pid-air-quality-co2-value", "pid-air-quality-co2-value-text"));
+        mapDescObj.put("0x061C", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_INTEGER, CATEGORY_UNDEFINED, "", "",
+                "pid-air-quality-co-value", "pid-air-quality-co-value-text"));
+        mapDescObj.put("0x061D", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_INTEGER, CATEGORY_UNDEFINED, "", "",
+                "pid-air-quality-no2-value", "pid-air-quality-no2-value-text"));
+        mapDescObj.put("0x061E", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_INTEGER, CATEGORY_UNDEFINED, "", "",
+                "pid-air-quality-o3-value", "pid-air-quality-o3-value-text"));
+        mapDescObj.put("0x061F", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_INTEGER, CATEGORY_UNDEFINED, "", "",
+                "pid-air-quality-pm10-value", "pid-air-quality-pm10-value-text"));
+        mapDescObj.put("0x0620", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_INTEGER, CATEGORY_UNDEFINED, "", "",
+                "pid-air-quality-pm25-value", "pid-air-quality-pm25-value-text"));
+        mapDescObj.put("0x0621", createFreeAtHomePairingIdTranslation(PID_VALUETYPE_INTEGER, CATEGORY_UNDEFINED, "", "",
+                "pid-air-quality-voc-value", "pid-air-quality-voc-value-text"));
+
+        MAP_TRANSLATOR = Collections.unmodifiableMap(mapDescObj);
+    }
```

Comment:
Can the creation of this Map be simplified using the same approach as in `FidTranslationUtils`?

Reply:
**Fixed**